source/reference/operator/aggregation/geoNear.txt

```diff
@@ -204,8 +204,147 @@ When using :pipeline:`$geoNear`, consider that:
 - Starting in version 4.2, :pipeline:`$geoNear` no longer has a default
   limit of 100 documents.
 
-Example
--------
+- Starting in MongoDB 5.1, the ``near`` parameter supports the 
+  :ref:`let option <geoNear_let_example>` and
+  :ref:`bound let option <geoNear_bounded_let_example>`.
+
+Examples
+--------
+
+.. _geoNear_let_example:
+
+$geoNear with the ``let`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example:
+
+- The ``let`` option is used to set an array value of 
+  ``[-73.99279,40.719296]`` to the variable ``$pt``.
+
+- ``$pt`` is a let option to the ``near`` parameter to the ``$geoNear`` 
+  stage.
+
+.. code-block:: javascript
+   :emphasize-lines: 6,16
+
+   db.places.aggregate(
+   [
+      {
+         "$geoNear":
+         {
+            "near":"$$pt",
+            "distanceField":"distance",
+            "maxDistance":2,
+            "query":{"category":"Parks"},
+            "includeLocs":"dist.location",
+            "spherical":true
+         }
+      }
+   ],
+   {
+      "let":{ "pt": [ -73.99279, 40.719296 ] }
+   }
+   )
+
+The aggregation returns the following:
+
+.. code-block:: javascript
+   :copyable: false
+
+   {
+      _id: ObjectId("61715cf9b0c1d171bb498fd7"),
+      name: 'Sara D. Roosevelt Park',
+      location: { type: 'Point', coordinates: [ -73.9928, 40.7193 ] },
+      category: 'Parks',
+      distance: 1.4957325341976439e-7,
+      dist: { location: { type: 'Point', coordinates: [ -73.9928, 40.7193 ] } }
+   },
+   {
+      _id: ObjectId("61715cf9b0c1d171bb498fd6"),
+      name: 'Central Park',
+      location: { type: 'Point', coordinates: [ -73.97, 40.77 ] },
+      category: 'Parks',
+      distance: 0.0009348548688841822,
+      dist: { location: { type: 'Point', coordinates: [ -73.97, 40.77 ] } }
+   }
+
+.. _geoNear_bounded_let_example:
+
+$geoNear With Bound ``let`` Option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``let`` option can bind a variable which can be used in a 
+$geoNear query.
+
+In this example, :pipeline:`$lookup` uses:
+
+- ``let`` to define ``$pt``.
+- :pipeline:`$geoNear` in the ``pipeline``.
+- ``$pt`` to define ``near`` in the :pipeline:`$geoNear` pipeline stage.
+
+.. code-block:: javascript
+   :emphasize-lines: 7
+
+   db.places.aggregate(
+   [
+   {
+      $lookup: 
+         {
+            from: "places",
+            let: { pt: "$location" },
+            pipeline: [
+               {
+                  $geoNear:
+                     {
+                        near: "$$pt",
+                        distanceField: "distance"
+                     }
+               }
+               ],
+            as: "joinedField"
+         }
+      },
+      { $match: { name: "Sara D. Roosevelt Park" } }
+   ]
+   );
```

Comment:
The indentation here is a little off. We (generally) need the closing brackets to align with their opening counterparts. Suggest something like:

```suggestion
   db.places.aggregate( [
   {
      $lookup: {
         from: "places",
         let: { pt: "$location" },
         pipeline: [
            {
               $geoNear: {
                  near: "$$pt",
                  distanceField: "distance"
               }
            }
         ],
         as: "joinedField"
      }
   },
   { 
      $match: { name: "Sara D. Roosevelt Park" }
   }
] );
```

Reply:
Should the { (previously line #290) for $lookup also come down to the next line to match the indent? See updated.